client/src/telemetry/glean-context.tsx

```diff
@@ -192,12 +197,18 @@ export function useGleanPage(pageNotFound: boolean) {
       viewportHorizontalCoverage: Math.round(
         (100 * window.innerWidth) / window.screen.width
       ),
+      isBaseline:
+        doc?.baseline?.is_baseline === undefined
+          ? undefined
+          : doc.baseline.is_baseline
+          ? "baseline"
+          : "not_baseline",
```

Comment:
Unfortunately typescript makes this less fun, as I have to manually cast `doc?.baseline?.is_baseline` to a string:
```js
      isBaseline: { true: "baseline", false: "no_baseline" }[
        String(doc?.baseline?.is_baseline)
      ],
```
So I'm going to leave as it is, especially since I imagine we'll need a second layer of logic for future the almost-baseline case, but I'll remember this for the future!